extensions/wikia/Wall/VoteHelper.class.php

```diff
@@ -52,162 +49,151 @@ function getVotersList( $from = 0, $count = 50 ) {
 
 		$this->getVoteCount();
 
-		wfProfileOut( __METHOD__ );
 		return $out;
 	}
 
 	function addVote( $score = 1 ) {
-		wfProfileIn( __METHOD__ );
 		if ( $this->isVoted() ) {
-			wfProfileOut( __METHOD__ );
 			return false;
 		}
 		$dbr = wfGetDB( DB_MASTER );
 
-		$values = array();
-		$values['article_id'] = $this->pageId;
-		$values['ip'] = $this->userIP;
-		$values['user_id'] = $this->userId;
-
-		$values['unique_id'] = md5( $this->userIP ); // Backward compatibility
-		$values['time'] = wfTimestampNow();
-		$values['vote'] = $score;
-
-		$dbr->insert( 'page_vote',
+		$values = [
+			'article_id' => $this->pageId,
+			'ip' => $this->userIP,
+			'user_id' => $this->userId,
+			'unique_id' => md5( $this->userIP ), // Backward compatibility
+			'time' => wfTimestampNow(),
+			'vote' => $score,
+		];
+
+		$dbr->insert(
+			'page_vote',
 			$values,
-		__METHOD__ );
+			__METHOD__
+		);
 
 		$cache = $this->getCache();
-		$cache->set( $this->getUserCacheKey(), 'Y', 60 * 60 );
+		$cache->set( $this->getUserCacheKey(), 'Y' );
```

Comment:
There are memcached updates for updates and deletes and we should keep this data as long as it can stay before getting pushed out of cache.